Docs: Add documentation for Custom Branding on Public Dashboards (#58090)
grafana/grafana: docs/sources/setup-grafana/configure-grafana/configure-custom-branding/index.md

- Grafana version with build number

If you specify `footer_links` or `GF_WHITE_LABELING_FOOTER_LINKS`, then all other default links are removed from the footer and only what is specified is included.

## Custom branding for Public Dashboards

> In addition to the customizations described below, as an Enterprise user, you can customize the footer of your public dashboards.

Reviewer: ```suggestion
In addition to the customizations described below, you can customize the footer of your public dashboards.
```

> In addition to the customizations described below, as an Enterprise user, you can customize the footer of your public dashboards.

Reviewer: just fyi that I am removing reference to the user being an Enterprise customer because this the Custom branding topic only applies to Enterprise customers. So it's redundant.

Author: Excelent! Thank you!

> In addition to the customizations detailed below, as an Enterprise user you can also customize the footer for your public dashboards.

Reviewer: ```suggestion
In addition to the customizations described below, as an Enterprise user, you can customize the footer of your public dashboards.
```

> To do this, you can add a new section on your `grafana.ini` file.

Reviewer: ```suggestion
To customize the footer of a public dashboard, add the following section to the `grafana.ini` file.
```

```ini
[white_labeling.public_dashboards]

# Hides the footer for the public dashboards if set to "true". If
# example: footer_hide = "true"
;footer_hide =

# Set to text shown in the footer
;footer_text =

# Set to complete url to override public dashboard footer logo
;footer_logo =

# Set to link for the footer
;footer_link =
```

> If you specify `footer_hide` to `true`, all the other values are ignored as the footer will not be shown.

Reviewer: ```suggestion
If you specify `footer_hide` to `true`, all the other values are ignored because the footer will not be shown.
```
